ostree-backup.sh

```diff
@@ -68,6 +71,12 @@ else
     echo "Skipping etc backup as it already exists"
 fi
 
+if [[ ! -f /var/tmp/backup/ostree.tgz ]]; then
+    tar czf /var/tmp/backup/ostree.tgz --selinux -C /ostree/repo .
```

Comment:
This one looks strange...oh, is this to handle kernel-rt type stuff where we have local overrides?

Reply:
it is a bit ugly, but it is a way that allows us for:
- rpm-ostree status output for the new deployment looks exactly as in the original cluster
- Preserve selinux attributes
- No need for extra os image when there were changes to the os level (in our case, the rt-kernel)
- Use ony one image as the artifact containing everything needed for the restore (var, etc, os, list of images to precache, rpm-ostree status in the original cluster, etc etc), having everything tied together in a single place

Reply:
@cgwalters other suggestion?